tools/ci/check_grep.sh

```diff
@@ -65,6 +65,11 @@ if grep -Pzo '"\w+" = \(\n[^)]*?/obj/structure/cable,\n[^)]*?/obj/structure/cabl
     echo -e "${RED}ERROR: Found multiple cables on the same tile, please remove them.${NC}"
     st=1
 fi;
+if grep -Pzo '"\w+" = \(\n[^)]*?/turf,\n[^)]*?/turf,\n[^)]*?/area/.+?\)' _maps/**/*.dmm;	then
+	echo
+    echo -e "${RED}ERROR: Found multiple /turfs on the same tile. We do not support turf stacking, please remove them.${NC}"
+    st=1
+fi;
```

Comment:
We already have this type of check in lines 175-178 (on master) or 181-184 (on your branch). Does that one not work?

Reply:
oh you do? ninja mentioned it like we didn't but I thought we did, I'll nuke it

Reply:
I coded it in two months ago after someone forgot to run the mapmerge scripts and it fucked up because StrongDMM allows you to place multiple turfs since it is valid in Gooncode.